docs/openapi3.yaml

```diff
@@ -1010,6 +1016,194 @@ paths:
               schema:
                 $ref: "#/components/schemas/error_response"
 
+  # Processors
+  /v1/processors:
+    post:
+      operationId: create-processor-file
+      tags:
+      - Processors
+      summary: Create processor file
+      description: Create a new processor file which will have javascript code that can be executed by tests.
+      responses:
+        '201':
+          description: Success
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/processor'
+        '400':
+          description: Bad request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/error_response'
+        '500':
+          description: Internal server error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/error_response'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/processor'
+        description: The processor file to create
+        required: true
+    get:
+      operationId: retrieve-all-processor-files
+      tags:
+      - Processors
+      summary: Retrieve all processors files
+      description: Retrieve the details of all processor files.
+      parameters:
+      - in: query
+        name: from
```

Comment:
are these needed? i feel like the amount of `processor files` created is limited and not sure how useful the query params are